.env.example

```
@@ -216,3 +216,13 @@ INTENT_LAYER_SHADOW_ENABLED_PROD=false
 # Fraction (0-100) of the eligible slice to mirror. Default 1 (1%). Deterministic per
 # correlation id. Set 0 to mirror nothing while leaving the flag on.
 INTENT_LAYER_SHADOW_SAMPLE_PCT=1
+
+
+# === BI Dashboard (GAP-043 L-bi — Superset) ===
+# Apache Superset 4.1.1 self-hosted analytics
+# Use: docker compose -f docker/docker-compose.bi.yml up -d
+# Access: http://localhost:8088 (admin/admin by default)
+SUPERSET_SECRET_KEY=change-me-in-production-use-256-bit-key
+# Superset admin credentials (first-time setup only; change in production)
+SUPERSET_ADMIN_USERNAME=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPERSET_ADMIN_USERNAME key should go before the SUPERSET_SECRET_KEY key

(UnorderedKey)